charts/spinnaker/templates/configmap/halyard-config.yaml

```diff
@@ -132,6 +147,18 @@ data:
                 {{ if $account.readPermissions }}--read-permissions {{ join " --read-permissions " $account.readPermissions }}{{ end }} \
                 {{ if $account.writePermissions }}--write-permissions {{ join " --write-permissions " $account.writePermissions }}{{ end }} \
                 --provider-version v2
+
+    {{- range $index, $customResource := $.Values.kubeConfig.customResources }}
+    if [ ! custom_resource_exists "${ACCOUNT}" '{{ $customResource.kubernetesKind }}' '{{ $customResource.spinnakerKind }}' ]; then
```

Comment:
This is not correct I believe. `[ ... ]` is essentially a shorthand for `test ...`, in which this case it's like `test -n ...`. In other words, it's not actually calling the function.

See:

```sh
❯ function hello() { echo 'hello'; }
❯ hello
hello
❯ if [ hello ]; then echo 'world'; fi
world
❯ if hello; then echo 'world'; fi
hello
world
```